### DISCUSSION TITLE
fatal error: 'fitz.h' file not found on freebsd

### DISCUSSION THREAD

**Discussion by Vitz** [pymupdf/PyMuPDF#992] [Looking for help]
## Describe the bug (mandatory)
Cant install using PIP
error:
>     fitz/fitz_wrap.c:2767:10: fatal error: 'fitz.h' file not found
>     #include <fitz.h>
>              ^~~~~~~~
>     1 error generated.
>     error: command 'cc' failed with exit status 1

## To Reproduce (mandatory)
Just using this command: pip install PyMuPDF

## Your configuration (mandatory)
```
$  cat /etc/os-release
NAME=FreeBSD
VERSION=12.2-RELEASE

```
`PyMuPDF 1.18.10  and previous `
`Python 3.6.12`

## Additional context (optional)
1. I can't install anything, because its django hosting with ssh, not a VPS.
2. Is that a chance it would work if I compile this package using FreeBSD?

**Comment by JorjMcKie** [comment]
> Is that a chance it would work if I compile this package using FreeBSD?

Presumably yes. As you can see in ``setup.py``, it handles this analogous to MacOSX. So - after installing MuPDF locally too - it should build PyMuPDF.
But don't ask me how the subsequent upload to your cloud might work.

BTW: if it is a bug, it is not a PyMuPDF bug. So I will close this issue and transfer it to "Discussions" (category "Asking for Help"), where you will also find similar postings around how to upload Python packages with binary extensions to clouds.

**Reply by Vitz** [reply]
Ok, I will try to compile it. I just dont wanna change the library, because it works great on windows.

Thank you for quick answer, 
have a nice day
